app/main.py

```diff
@@ -1,3 +1,61 @@
 class Distance:
     # Write your code here
-    pass
+    def __init__(self, km: float):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if not isinstance(other, Distance):
```

Comment:
you can write static function which will check type of other. so you won't need to write additional isinstance in every (or almost every) function